src/components/card/Productcard.jsx

```diff
@@ -1,38 +1,56 @@
-import React from 'react';
-import { useCart } from '../../context/cart-context';
-import { useWishlist } from '../../context/wishlist-context';
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
 
-const Productcard = ({product}) => {
-
-  const {addToCart} = useCart();
-  const {addToWishlist} = useWishlist();
+const Productcard = ({ product }) => {
+  const { addToCart } = useCart();
+  const { addToWishlist } = useWishlist();
   return (
-   <>
-   <div className="gh-card card card-image shadow">
+    <>
+      <div className="gh-card card card-image shadow">
         <div className="card-image-container">
           <i className="ph-heart card-icon-heart gh-heart"></i>
-          <img src={product.imgSrc} alt="product-image" className="card-image" />
+          <img
+            src={product.imgSrc}
```

Comment:
you can destructure this where it repeats again and again like the **product.imgsrc product.name** etc